CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## NEXT
 
+## Fixed
+* **BREAKING**: Dropdown voorbeeld met `btn-default` button gebruikt verkeerde markup (#957)

```

Comment:
Pas je de notatie nog even aan:
- we geven aan op welke van de packages de change betrekking heeft;
- we maken de issue/pr referenties linkend;
- bij een breaking change refereren we aan de PR alwaar we dan vertellen wat een developer zou moeten doen, of anderszins een toelichting.